src/compiler/preprocess/index.ts

```diff
@@ -83,6 +83,80 @@ async function replace_async(
 	return out.concat(final_content);
 }
 
+
+
+// import decoded sourcemap from mozilla/source-map/SourceMapGenerator
```

Comment:
you might as well make this in the jsdoc style in case we were ever to generate docs from the source code one day

```
/**
 *
 */
```